### ISSUE TITLE
Support zone.js 0.9

### ISSUE THREAD

**Issue by fr0** [angular/angular#29570]
# 🚀 feature request

### Relevant Package

 This feature request is for @angular/core

### Description
 A clear and concise description of the problem or missing capability...

When using the latest version of zone.js, `0.9.0` with the latest version of `@angular/core`, you get a warning on install:

```
warning " > @angular/core@7.2.11" has incorrect peer dependency "zone.js@~0.8.26".
```

### Describe the solution you'd like

I'd like Angular to support the latest version of zone.js without any warnings.

**Comment by jotatoledo** [comment]
Any reason for it? do you need features from zone.js@0.9 on the current stable version of angular?

Btw, next already updated to zone.js https://github.com/angular/angular/pull/28219

**Comment by pkozlowski-opensource** [comment]
> : Btw, next already updated to zone.js@0.9 #28219

Yeh, this is already fixed in master

**Comment by fr0** [comment]
Angular 7.2.14 still has this issue. When is the fix going to be released?

**Comment by gkalpak** [comment]
Since the update from 0.8.x to 0.9.x implies breaking changes, it has only been merged into the `@next` version and not the `@latest` (stable) one (7.x). It was released in 8.0.0-beta.10.
